.github/workflows/article_combiner_quality.yml

```diff
@@ -0,0 +1,43 @@
+name: Test and Lint article_combiner
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+permissions: write-all
+
+jobs:
+  quality_check:
+    name: pytest and pylint
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout the code
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    # Install Python
+    - name: Install Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.12'
+
+    # Install required packages
+    - name: Install packages
+      run: |
+        pip install pylint pytest pytest--cov
+        pip install -r ./article_combiner/requirements.txt
```

Comment:
this wouldn't fail based on the coverage would it? just letting us know?

Reply:
No i don't believe so since the tests passed for this PR. I assume the coverage is purely information.